db/create_tables.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE poll(
+    id NUMERIC,
+    title VARCHAR(20) NOT NULL,
+    vote_until DATE DEFAULT CURRENT_DATE + INTERVAL '10 day',

```

Comment:
締切に date 型が使われているのですが、12時締切の場合や17時締切の場合というのも考えられるので timestamp 型を使用した方が無難そうです。
https://www.postgresql.jp/document/12/html/datatype-datetime.html

```suggestion
    vote_until timestamp,
```